### PR TITLE
Update yolox_loss.py

### PR DESCRIPTION
**Pull request by eran-deci**

### CODE REVIEW

File: src/super_gradients/training/losses/yolox_loss.py [hunk 1]
```diff
@@ -112,9 +112,24 @@ class YoloXDetectionLoss(_Loss):
...
+    :param iou_weight:              Weight to apply to the iou loss term.
+    :param obj_weight:              Weight to apply to the obj loss term.
+    :param cls_weight:              Weight to apply to the cls loss term.
+    :param cls_pos_weight:          Class weights for the cls loss. Passed on to torch.nn.BCEWithLogitsLoss
     """
 
-    def __init__(self, strides: list, num_classes: int, use_l1: bool = False, center_sampling_radius: float = 2.5, iou_type: str = "iou"):
+    def __init__(
+        self,
+        strides: list,
```
Comment (BloodAxe) [inline src/super_gradients/training/losses/yolox_loss.py:123]: Type annotation should be `List[int]` from typing and not built-in `list`.

**Review by BloodAxe — COMMENTED** [review]
Looks good! Just small typing annotation comment and we can merge this PR

File: src/super_gradients/training/losses/yolox_loss.py (hunk 1 above)
Comment (eran-deci) [reply src/super_gradients/training/losses/yolox_loss.py]: fixed

**Comment by eran-deci** [comment]
Thanks for the review @BloodAxe 
I addressed the typing annotation issue

**Review by BloodAxe — APPROVED** [review]
LGTM

**Pull request merged**